components/layout/header.vue

```diff
@@ -18,21 +43,203 @@
+              />
+            </PopoverButton>
+
+            <transition
+              enter-active-class="transition ease-out duration-200"
+              enter-from-class="opacity-0 translate-y-1"
+              enter-to-class="opacity-100 translate-y-0"
+              leave-active-class="transition ease-in duration-150"
+              leave-from-class="opacity-100 translate-y-0"
+              leave-to-class="opacity-0 translate-y-1"
+            >
+              <PopoverPanel
+                class="absolute z-10 -ml-4 mt-3 transform px-2 w-screen max-w-md sm:px-0 lg:ml-0 lg:left-1/2 lg:-translate-x-1/2"
+              >
+                <div
+                  class="rounded-lg shadow-lg ring-1 ring-black ring-opacity-5 overflow-hidden"
+                >
+                  <div
+                    class="relative grid gap-6 bg-white px-5 py-6 sm:gap-8 sm:p-8"
+                  >
+                    <a
+                      v-for="item in solutions"
+                      :key="item.name"
+                      :href="item.href"
+                      class="-m-3 p-3 flex items-start rounded-lg hover:bg-gray-50"
+                    >
+                      <component
+                        :is="item.icon"
+                        class="flex-shrink-0 h-6 w-6 text-indigo-600"
+                        aria-hidden="true"
+                      />
+                      <div class="ml-4">
+                        <p class="text-base font-medium text-gray-900">
+                          {{ item.name }}
+                        </p>
+                        <p class="mt-1 text-sm text-gray-500">
+                          {{ item.description }}
+                        </p>
+                      </div>
+                    </a>
+                  </div>
+                  <div
+                    class="px-5 py-5 bg-gray-50 space-y-6 sm:flex sm:space-y-0 sm:space-x-10 sm:px-8"
+                  >
+                    <div
+                      v-for="item in callsToAction"
+                      :key="item.name"
+                      class="flow-root"
+                    >
+                      <a
+                        :href="item.href"
+                        class="-m-3 p-3 flex items-center rounded-md text-base font-medium text-gray-900 hover:bg-gray-100"
+                      >
+                        <component
+                          :is="item.icon"
+                          class="flex-shrink-0 h-6 w-6 text-gray-400"
+                          aria-hidden="true"
+                        />
+                        <span class="ml-3">{{ item.name }}</span>
+                      </a>
+                    </div>
+                  </div>
+                </div>
+              </PopoverPanel>
+            </transition>
+          </Popover>
+
+          <a
+            href="#"
             class="text-base font-medium text-gray-500 hover:text-gray-900"
-            v-for="item in navigation"
```

Comment:
use navigation array to define navigation. see the code that was already there ;)

Reply:
Have a look now